apps/polyflix/src/modules/collections/components/CollectionTimeline/ItemCollectionTimeline.component.tsx

```diff
@@ -116,7 +117,7 @@ export const ItemCollectionTimeline = ({
                 variant="body1"
                 sx={{
                   textDecoration: isHover ? 'underline' : 'none',
-                  color: 'black',
+                  color: +query.get('index')! === index ? 'red' : 'grey',
```

Comment:
```suggestion
                  color: (theme) => +query.get('index')! === index ? 'theme.color.primary.main : 'theme.color......',
```
or theme.text // maybe it change auto with the theme